deploy/template.yaml

```diff
@@ -92,10 +92,10 @@ Mappings:
       tg500ErrorLimit: 10
       tg500ErrorWindow: 300
       lambdaInvokeCompareWindow: 300
-      fargateCPUsize: "256"
-      fargateRAMsize: "512"
-      nodeOldSpaceLimit: "384"
-      nodeEnv: "development"
+      fargateCPUsize: "1024"
```

Comment:
can we make sure this is the minimum across all envs plz.

Reply:
This is not a genuine PR - this is only for the dev env testing